src/models/users.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+export interface IUser extends Document {
+  name: string;
+  email: string;
+  password: string;
+  isDisabled: boolean;
+  createdDate: Date;
+  updatedDate: Date;
+}
+
+const UserSchema: Schema = new Schema({
+  name: { type: String, required: true },
+  email: { type: String, required: true },
+  password: { type: String, required: true },
+  isDisabled: { type: Boolean, required: true, default: false },
+  createdDate: { type: Date, default: Date.now },
```

Comment:
Instead of defining createdDate and updatedDate Use it like below it will handle dates automatically.

timestamps: { createdAt: 'createdDate', updatedAt: 'updatedDate' }

Reply:
Done